packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/stop-bot-modal.tsx

```diff
@@ -7,37 +7,48 @@ import { isMobile } from '@deriv/shared';
 type TStopBotModal = {
     is_running: boolean;
     is_dialog_open: boolean;
+    is_contract_dialog_open: boolean;
+    is_stop_bot_dialog_open: boolean;
     onOkButtonClick: () => void;
     toggleStopBotDialog: () => void;
+    stopMyBot: () => void;
+    closeMultiplierContract: () => void;
+    is_multiplier: boolean;
 };
 
-const StopBotModalContent = ({ is_running, is_dialog_open, onOkButtonClick, toggleStopBotDialog }: TStopBotModal) => {
-    const confirm_button_text = is_running ? localize('Stop my bot') : localize('Keep my contract');
-    const cancel_button_text = is_running ? localize('Back') : localize('Close my contract');
-    const title_text = is_running ? localize('Stop your current bot?') : localize('Keep your current contract?');
+const StopBotModalContent = ({
+    is_running,
+    onOkButtonClick,
+    toggleStopBotDialog,
+    is_contract_dialog_open,
+    is_stop_bot_dialog_open,
+    is_multiplier,
```

Comment:
here the same, would be better to keep in the order we usually keep: first the variables, then the functions

Reply:
This is something we also need to automate from eslint. It should order this props accordingly.

Reply:
plus check please others places

Reply:
This is done ✅